Add documentation for supported probes (#8)
probe-rs/webpage: md/udev.md

# udev rules

By default, the debug probes are only accessible by users with root privileges on Linux based systems.

> It is recommend to use appropriate udev rules so that also users without root priviliges have
> access to the debug probes.

Reviewer: ```suggestion
It is recommend to use appropriate udev rules such that users without root privileges too have
access to the debug probes.
```
